code/modules/power/power_machine.dm

```diff
@@ -13,38 +13,38 @@
 /obj/machinery/power/Destroy()
 	disconnect_from_network()
 	return ..()
-
+// Produce power (Watts)
 /obj/machinery/power/proc/produce_direct_power(amount)
 	if(powernet)
 		powernet.queued_power_production += amount
 		return TRUE
 	return FALSE
 
-/// Adds power demand to the powernet, machines should use this
+/// Adds power demand to the powernet, machines should use this (Watts)
 /obj/machinery/power/proc/consume_direct_power(amount)
 	powernet?.power_demand += amount
 
-/// Gets surplus power available on this machines powernet, machines should use this proc
+/// Gets surplus power available on this machines powernet, machines should use this proc (Watts)
```

Comment:
and here & down. `(Watts)` doesn't really help with explaining this proc better, instead you should note that the amount given in params is treated as X Watts

Reply:
I'm using this format to keep things concise. The idea is to be able to tell at a glance which units each proc and variable use. Besides that, these procs mostly one liners, and fairly self explanatory as it is.

I did rearrange the comment to make it clearer what "(Watts)" refers too, I believe this should be enough.